setup.cfg

```diff
@@ -1,3 +1,6 @@
 [flake8]
 max-line-length = 99
-ignore = W504
\ No newline at end of file
+
+ignore = 
+    W504,  # W504: Line break occurred after a binary operator
+    E741  # E741: Do not use variables named 'I', 'O', or 'l'
```

Comment:
Can you create an issue to fix the `E741` errors on github, and in a follow up PR add a comment above this line with the issue number please.